neqo-http3/src/connection.rs

```diff
@@ -1133,16 +1133,23 @@ mod tests {
         };
     }
 
+    #[derive(PartialEq)]
+    enum Role {
+        Client,
+        Server,
```

Comment:
Can we use neqo_transport::Role here? It is pretty simple and might also serve here.

Reply:
removed.